core/requirements.txt

```diff
@@ -0,0 +1,3 @@
+click==6.7
+-e git+https://github.com/SecurityInnovation/PGPy.git@release/0.4.1#egg=PGPy-0.4.1
```

Comment:
i like to give possibilities for installing in different ways. What's wrong with it?.